app/components/floating-button/FloatingButton.js

```diff
@@ -37,6 +37,10 @@ class FloatingButton extends Component {
 
   handleClose = () => this.setState({ isShowingActions: false })
 
+  showComposer(type) {
+    return this.props.showComposer.bind(this, type)
```

Comment:
This seems like a pretty elegant solution to me, but afaik you'd still be getting the performance hit from creating a new function on the fly every time, which is what the eslint rule was trying to prevent.
I can't immediately think of a way around it though because we can't bind a method on `this.props` in the constructor..